### PR TITLE
Create issue_template.md

### PR DESCRIPTION
**Pull request by VictorAlagwu**

### CODE REVIEW

File: issue_template.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## Expected Result
```
Comment (SD10) [inline issue_template.md]: Maybe, ## What happened?

File: issue_template.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+
+
+
+## Actual Result Gotten
```
Comment (SD10) [inline issue_template.md]: Maybe, ## What did you expect to happen?

File: issue_template.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+
+
+## Ways to Resolve this issue
```
Comment (SD10) [inline issue_template.md]: Probably don't need this section

File: issue_template.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+
+
+
+## Information about Device Issue was experience
```
Comment (SD10) [inline issue_template.md]: ## Additional Information
- [ ] MessageKit Version:
- [ ] iOS Version(s):
- [ ] Swift Version:
- [ ] Devices/Simulators:
- [ ] Reproducible in ChatExample? (Yes/No):

**Comment by SD10** [comment]
@VictorAlagwu Thank you for getting started on this! 👍  I made some comments describing what I had in mind. 

I'm thinking maybe we should rename `Additional Information` to `General Information` and put it at the very start of the issue template.

**Comment by VictorAlagwu** [comment]
I just added a new commit

**Comment by VictorAlagwu** [comment]
or let me close this

**Pull request closed without merging**